tests/Fixer/Basic/BracesPositionFixerTest.php

```diff
@@ -312,35 +312,6 @@ public static function provideFixCases(): iterable
             ['control_structures_opening_brace' => 'next_line_unless_newline_at_signature_end'],
         ];
 
-        yield 'try catch finally (default)' => [
-            '<?php
-                switch ($foo) {
-                    case 1:
-                        foo();
-                }',
-            '<?php
-                switch ($foo)
-                {
-                    case 1:
-                        foo();
-                }',
-        ];
-
-        yield 'try catch finally (next line)' => [
-            '<?php
-                switch ($foo)
-                {
-                    case 1:
-                        foo();
-                }',
-            '<?php
-                switch ($foo) {
-                    case 1:
-                        foo();
-                }',
-            ['control_structures_opening_brace' => 'next_line_unless_newline_at_signature_end'],
-        ];
-
```

Comment:
```suggestion
        yield 'try catch finally (default)' => [
            '<?php
                try {
                    foo();
                }
                catch (Exception $e) {
                    bar();
                }
                finally {
                    baz();
                }',
            '<?php
                try
                {
                    foo();
                }
                catch (Exception $e)
                {
                    bar();
                }
                finally
                {
                    baz();
                }',
        ];

        yield 'try catch finally (next line)' => [
            '<?php
                try
                {
                    foo();
                }
                catch (Exception $e)
                {
                    bar();
                }
                finally
                {
                    baz();
                }',
            '<?php
                try {
                    foo();
                }
                catch (Exception $e) {
                    bar();
                }
                finally {
                    baz();
                }',
            ['control_structures_opening_brace' => 'next_line_unless_newline_at_signature_end'],
        ];
```

These cases were copied and descriptions were updated, but content remained the same. We should modify them to test what they were supposed to test - `try/catch/finally`.

Reply:
I have to do it manually, because:
> Applying suggestions on deleted lines is currently not supported.

Reply:
Yeah, I know, I made suggestion manually too 😅. Just for visual diff.